symphony/app/fbcnms-projects/hub/app/components/Main.js

```diff
@@ -109,8 +109,9 @@ function Main(subApp) {
 
 export default () => {
   const relativeUrl = useRelativeUrl();
+  const props = {adminMode: true}; //TODO verify
   const cs = () => Main(CreateServiceForm());
-  const wf = () => Main(WorkflowApp());
+  const wf = () => Main(WorkflowApp(props));
```

Comment:
I would create a new App component in favor of introducing boolean params.

So instead of sprinkling conditions in existing components we would compose new ones.
E.g. instead of displaying controls based on conditions  " (props.adminMode ? '/metadata' : '/metadata'); " we would just not import/render them at all in the new components